blackberggroup/VA-OHIL
# Feat/jls 7126754806/add iex banner

## What type of PR is this? (check all applicable)


## Description
Develop iEX 2024 banner under hero based upon Figma design; open to breakpoint designs

## Related Tickets & Documents
- Monday Task https://blackberg-group.monday.com/boards/1931719392/pulses/7126754806

## QA Instructions, Screenshots, Recordings
Confirm page looks like Figma design
Ensure links (image, hotel, CTA) all go to to registration page
Major breakpoint at tablet-lg: ensure there's enough padding between banner elements

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _includes/home/iex-banner.html
 - _site/assets-3-5/img/home/iex-2024-logo.webp
 - _site/assets/img/home/iex-2024-logo.webp
 - _site/assets/sass/vha/pages/home/iex-banner/_index.scss
 - assets/img/home/iex-2024-logo.webp
 - assets/sass/vha/pages/home/iex-banner/_index.scss
 - _includes/footer.html
 - _includes/home/hero.html
 - _site/assets-3-5/css/styles.css
 - _site/assets/css/styles.css
 - _site/assets/sass/vha/pages/home/_index.scss
 - _site/assets/sass/vha/pages/home/hero/_index.scss
 - _site/index.html
 - assets/css/styles.css
 - assets/sass/vha/pages/home/_index.scss
 - assets/sass/vha/pages/home/hero/_index.scss
 - index.html
